build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
+
+mkdir -p bin
+cd bin
+
+go build ../cmd/...
```

Comment:
`go build -o ./bin/openshift-install ./cmd/openshift-install` instead of `cd bin`